jquery.oembed.js

```diff
@@ -54,49 +54,56 @@
+                // The service does not support https, so if short url is not in providers it will not work
+                if (!provider || provider.name === "opengraph" && window.location.protocol !== "https:") {
+                    for (var j = 0, l = shortURLList.length; j < l; j++) {
+                        var regExp = new RegExp('://' + shortURLList[j] + '/', "i");
+
+                        if (resourceURL.match(regExp) !== null) {
+                            //AJAX to //api.longurl.org/v2/expand?url=//bit.ly/JATvIs&format=json&callback=hhh
+                            var ajaxopts = $.extend({
+                                url: "http://api.longurl.org/v2/expand",
+                                dataType: 'jsonp',
+                                data: {
+                                    url: resourceURL,
+                                    format: "json"
+                                    //callback: "?"
+                                },
+                                success: function (data) {
+                                    //this = $.fn.oembed;
+                                    resourceURL = data['long-url'];
+                                    provider = $.fn.oembed.getOEmbedProvider(data['long-url']);
+
+                                    //remove fallback
+                                    if (!!settings.fallback === false) {
+                                        provider = provider.name.toLowerCase() === 'opengraph' ? null : provider;
+                                    }
+
+                                    if (provider !== null) {
+                                        provider.params = getNormalizedParams(settings[provider.name]) || {};
+                                        provider.maxWidth = settings.maxWidth;
+                                        provider.maxHeight = settings.maxHeight;
+                                        embedCode(container, resourceURL, provider);
+                                    } else {
+                                        settings.onProviderNotFound.call(container, resourceURL);
+                                    }
+                                },
+                                error: function () {
+                                    settings.onError.call(container, resourceURL)
                                 }
-                            },
-                            error: function () {
-                                settings.onError.call(container, resourceURL)
-                            }
-                        }, settings.ajaxOptions || {});
+                            }, settings.ajaxOptions || {});
 
-                        $.ajax(ajaxopts);
+                            $.ajax(ajaxopts);
 
-                        return container;
+                            return container;
+                        }
                     }
+                    provider = $.fn.oembed.getOEmbedProvider(resourceURL);

```

Comment:
This line will never fire, you're already ensuring there isn't a `provider` based on the `resourceURL`. Safe to remove this.